make_level.py

```diff
@@ -0,0 +1,128 @@
+import sys
+import os
+
+cwd = os.getcwd()
+
+def write_init(skill_name, skill_path, level_name):
+    # If skills/skill/__init__.py doesn't exist, create a basic version (no levels)
+    if not os.path.exists(os.path.join(skill_path,"__init__.py")):
+        with open(os.path.join(skill_path,"__init__.py"), 'w+') as fp:
+            level_setup = "\n".join([
+                "import pkg_resources\n",
+                "from gitgud.skills.util import BasicLevel",
+                "from gitgud.skills.util import Skill",
+                "",
+                "skill = Skill(",
+                "    '{}',".format(skill_name),
+                "    [",
+                "    ]",
+                ")",
+                ""
+                ])
+            fp.write(level_setup)
+    
+        
+        # Read file
+        with open(os.path.join("gitgud","skills","__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        # Add import statement
+        replace = "\n".join([
+                            "from gitgud.skills.{0} import skill as {0}_skill".format(skill_name),
+                            "",
+                            "",
+                            "from gitgud.skills.u"
+                            ])
+        filedata = filedata.replace("\nfrom gitgud.skills.u", replace)
+        
+        # Add to input array of Skill
+        replace = ",\n    {}_skill\n]".format(skill_name)
+        filedata = filedata.replace("\n]", replace)
+
+        # Write to file
+        filepath = os.path.join("gitgud","skills","__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Registered skill \"{}\" in {}".format(skill_name, filepath))
+        
+        write_init(skill_name, skill_path, level_name)
+        return
+    else:
+        # Populate file with BasicLevel
+        with open(os.path.join(skill_path,"__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        replace = ",\n        BasicLevel('{0}', pkg_resources.resource_filename(__name__, '_{0}/'))\n    ]".format(level_name)
+        filedata = filedata.replace("\n    ]", replace)
+        filedata = filedata.replace("[,","[")
+        
+        filepath = os.path.join(skill_path,"__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Registered level \"{}\" in {}".format(level_name, filepath))
+    return
+
+def create_level_file(level_path,filename):
+    filepath = os.path.join(level_path,filename)
+    with open(filepath,'a+'):
+        pass
+    print("Created: {}".format(filepath))
+
+def main():
+    # Obtain input arguments
+    try:
+        skill_name = sys.argv[1]
+        level_name = sys.argv[2]
+    except:
+        print("Usage: \"python make_level.py <skill_name> <level_name>\"")
+        
+    # Check if current dir is ../gitgud directory. (i.e. dir of setup files)
+    if os.path.isdir(os.path.join(cwd, 'gitgud')):
```

Comment:
Instead of doing an if/else, I think it makes sense to do an if/return and keep the rest of the code unindented, so 
```
if not os.path.isdir(...):
    print(stuff)
    return
# Rest of the code below will go here and will have one less indent
```

Reply:
done